openlibrary/views/showmarc.py

```diff
@@ -106,19 +112,17 @@ def GET(self, filename, offset, length):
         offset = int(offset)
         length = int(length)
 
-        #print "record_locator: <code>%s</code><p/><hr>" % locator
+        # print("record_locator: <code>%s</code><p/><hr>" % locator)
 
         r0, r1 = offset, offset+100000
-        url = 'http://www.archive.org/download/%s'% filename
-
-        ureq = urllib.request.Request(url,
-                               None,
-                               {'Range':'bytes=%d-%d'% (r0, r1)},
-                               )
+        url = 'http://www.archive.org/download/%s' % filename
```

Comment:
@dherbst looks good! I just noticed the URLs in this file are old and take two redirects to get to the current:
`https://archive.org/...` URL

Reply:
```suggestion
        url = 'https://www.archive.org/download/%s' % filename
```

Reply:
@cdrini It needs to be  `url = 'https://archive.org/download/%s' % filename`  , `www` is a redirect also.